backend/src/db/migrations/8-user-sessions.sql

```diff
@@ -7,6 +7,9 @@ create table user_sessions (
     display_name text not null,
     roles text[] not null,
 
+    -- When the session was created. Always in UTC!
+    created timestamp not null default now(),
+
     -- When the session was last used. Always in UTC!
     last_used timestamp not null default now()
 );
```

Comment:
Probably just remove `last_used`. It's not used at all and was mainly intended for session timeouts.